client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -194,101 +171,25 @@ const InvestigationTable: React.FC = (): JSX.Element => {
-                                                    </a>
-                                                </Typography>
-                                            </div>
-                                        </>
-                                        :
-                                        ''
-                                )}
-                                inputValue={selectedInvestigator.userName}
-                                onChange={async (event, newSelectedInvestigator) => {
-                                    const result = await alertWarning(
-                                        `<p> האם אתה בטוח שאתה רוצה להחליף את החוקר <b>${indexedRow.investigatorName}</b> בחוקר <b>${newSelectedInvestigator?.value.userName}</b>?</p>`, {
-                                        showCancelButton: true,
-                                        cancelButtonText: 'לא',
-                                        cancelButtonColor: theme.palette.error.main,
-                                        confirmButtonColor: theme.palette.primary.main,
-                                        confirmButtonText: 'כן, המשך'
-                                    });
-                                    if (result.isConfirmed) {
-                                        indexedRow.groupId ?
-                                            await changeGroupsInvestigator([indexedRow.groupId], newSelectedInvestigator) :
-                                            await changeInvestigationsInvestigator([indexedRow.epidemiologyNumber], newSelectedInvestigator);
-                                        fetchTableData();
-                                    }
-
-                                    setSelectedRow(UNDEFINED_ROW);
-                                }}
-                                onInputChange={(event, selectedInvestigatorName) => {
-                                    if (event?.type !== 'blur') {
-                                        const updatedInvestigator = {
-                                            id: getUserMapKeyByValue(allUsersOfCurrCounty, selectedInvestigatorName),
-                                            userName: selectedInvestigatorName
-                                        }
-                                        setSelectedInvestigator(updatedInvestigator);
-                                    }
-                                }}
-                                renderInput={(params) =>
-                                    <TextField
-                                        {...params}
-                                        placeholder='חוקר'
-                                    />
-                                }
-                                classes={{
-                                    option: classes.userSelectOption
-                                }}
-                            />
-                        </div>
+                        <InvestigatorAllocation 
```

Comment:
You should change the name to `InvestigatorAllocationDialog` to be more clear.

Reply:
fixed